tiledb/api/src/stats.rs

```diff
@@ -0,0 +1,70 @@
+use crate::error::Error;
+use crate::Result as TileDBResult;
+
+pub fn stats_enable() -> TileDBResult<()> {
```

Comment:
I think this should probably be just `enable()` vs `stats_enable()`. Rust conventions for calling functions is to import the parent module which means if we have it as `stats_enable()` then it'd look something like:

```rust
use tiledb::stats;

fn foo() -> {
    stats::stats_enable();
}
```

instead of

```rust
use tiledb::stats;

fn foo() -> {
    stats::enable();
}
```